### PR TITLE
add pause/start schedule api

### PR DESCRIPTION
**Pull request by fewdan**
Signed-off-by: “fewdan” <fewdan@hotmail.com>

### What problem does this PR solve?

add pause/start schedule api

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by fewdan** [comment]
@g1eny0ung   
Need modify the UI, so PTAL.

**Review by YangKeao — APPROVED** [review]
LGTM

File: pkg/apiserver/schedule/schedule.go [hunk 1]
```diff
@@ -835,3 +838,130 @@ func (s *Service) batchDeleteSchedule(c *gin.Context) {
...
+	if err != nil {
+		_ = c.Error(utils.ErrInvalidRequest.WrapWithNoMessage(err))
+		return
+	}
+
+	uid := c.Param("uid")
+	if schedule, err = s.schedule.FindByUID(context.Background(), uid); err != nil {
+		if gorm.IsRecordNotFoundError(err) {
+			c.Status(http.StatusInternalServerError)
+			_ = c.Error(utils.ErrInvalidRequest.New("the experiment is not found"))
```
Comment (g1eny0ung) [inline pkg/apiserver/schedule/schedule.go:918]: the schedule is not found

File: pkg/apiserver/schedule/schedule.go [hunk 2]
```diff
@@ -835,3 +838,130 @@ func (s *Service) batchDeleteSchedule(c *gin.Context) {
 		c.JSON(http.StatusOK, StatusResponse{Status: "success"})
 	}
 }
+
+// @Summary Pause a schedule experiment.
```
Comment (g1eny0ung) [inline pkg/apiserver/schedule/schedule.go:842]: > Pause a schedule.

The same below, `schedule experiment` to `schedule`.

File: pkg/apiserver/schedule/schedule.go [hunk 3]
```diff
@@ -835,3 +838,130 @@ func (s *Service) batchDeleteSchedule(c *gin.Context) {
...
+		return
+	}
+
+	exp := &Base{
+		Name:      schedule.Name,
+		Namespace: schedule.Namespace,
+	}
+
+	annotations := map[string]string{
+		v1alpha1.PauseAnnotationKey: "false",
```
Comment (g1eny0ung) [inline pkg/apiserver/schedule/schedule.go:932]: If the `start` and `pause` function only has this line different, I think it's better to define a common function and then apply different params to it.

**Review by g1eny0ung — APPROVED** [review]

**Pull request merged**